test/test_runner.py

```diff
@@ -75,7 +82,7 @@ def assert_not_request(app, method, path):
 def test_execute(server, app):
     headers = {"Authorization": "Bearer 123"}
     execute(f"http://127.0.0.1:{server['port']}/swagger.yaml", api_options=dict(headers=headers))
-    assert len(app["saved_requests"]) == 2
+    assert len(app["saved_requests"]) >= 2
```

Comment:
How many exactly?

Reply:
3

Reply:
Please, set the exact number

Reply:
Next time `app` would be expanded it would affect these numbers in each test in this module :(

Reply:
At least it will be explicit